Ninja.cmake

```diff
@@ -28,9 +28,10 @@
 #
 # | CMake Variable          | Description                                                                                                               |
 # |-------------------------|---------------------------------------------------------------------------------------------------------------------------|
-# | NINJA_VERSION           | The version of Ninja to download - if it can't be found. Defaults to 1.10.2.                                              |
+# | NINJA_VERSION           | The version of Ninja to download - if it can't be found. Defaults to 1.11.0.                                              |
 # | NINJA_ARCHIVE_HASH      | The hash of the Ninja archive, following the format of `string(<HASH>`. Defaults to the hash of the Ninja 1.10.2 archive. |
 # | TOOLCHAIN_TOOLS_PATH    | The path to download tools to. If not set, then tools will not be downloaded.                                             |
```

Comment:
NOTE: different meaning for the same variable!

Reply:
Tracked as #50.